code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -202,6 +202,19 @@
 	desc = "A flask of holy water...it's been sitting in the Necropolis a while though."
 	list_reagents = list("hell_water" = 100)
 
+/obj/item/reagent_containers/food/drinks/holyoil
+	name = "flask of zelus oil"
+	desc = "A brass flask of the zelus oil."
+	icon_state = "zelusflask"
+	list_reagents = list("holyoil" = 30) //Powerfull
+	volume = 30
+	foodtype = NONE
+	force = 12 //Same as a toolbox
```

Comment:
this is pointless due to how drinks code totally overrides normal attack().

Reply:
Yep! Thats also intended.